src/helpers/reducedQueries.js

```diff
@@ -139,7 +139,7 @@ const hasVariable = (selectionSet, variable) => (
         const isVariableInArguments = selection.arguments.some(({ value }) => (
             value?.name?.value === variable
         ));
-        const isVariableInDirectives = selection.directives.some((directive) => (
+        const isVariableInDirectives = (selection.directives || []).some((directive) => (
             directive.arguments.some(({ value }) => value?.name?.value === variable)
```

Comment:
@mindnektar Or would it be wiser to also check if the directives arguments are also not an array?